packages/opentelemetry-scope-zone-peer-dep/README.md

```diff
@@ -0,0 +1,69 @@
+# OpenTelemetry Scope Zone Peer Dependency
+[![Gitter chat][gitter-image]][gitter-url]
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+This module provides *Zone Scope Manager with a peer dependency for [zone-js]* for Web applications.
```

Comment:
Could you mention that this package could be suitable for applications that already have Zone.js built in such as Angular applications?